### ISSUE TITLE
Report of all 10/20 service year award recipients

### ISSUE THREAD

**Issue by tskeeley** [CON-In-A-Box/CIAB-Signin#178]
Show a list of all concom members who should be getting a 10/20 year award.  Allow download to csv.  limit to admins (MIGHT add volunteer department eventually, but not now)

**Comment by AricStewart** [comment]
Right now the years served value for Concom is not valid as the data from neon is incomplete. We need the data before we can generate reports from it.

**Comment by AricStewart** [comment]
- Ok where would this live, Presumably under 'Concom List'
- Only usable by Directors? Heads? Who?

**Comment by tskeeley** [comment]
Concom list - admin function.  Directors and admins only.

**Comment by tskeeley** [comment]
I just needed to generate the list manually again, which made me remember this issue. :)  So, ping. :)

Is this something you'd prefer I work on?

**Comment by AricStewart** [comment]
I believe the last time I looked the mail issue was that all the data in question was not in the database. We only had a few years of concom info, so even if the facility was in place it would not function.
Is there a clean way to back-popuplate the data so that it would be worth while to implement?

**Comment by tskeeley** [comment]
We have all the data in spreadsheet form, and would much rather put it in the DB so we can stop manually tracking it.

**Comment by AricStewart** [comment]
Cool I see two possible ways to import this data...
We write a script/tool to read the spreadsheet and inserts it into then `ConComList` table. Though we would have to import all the conventions and convention years. We have the convention years, for conventions we have back to 2013. Not terrible but a bit more leg work.

Second we could add a new table for 'Historical years served'  and have it have data for pre-year XXXX and just add that value to whatever we calculated. 

I think I am much more of a fan of plan 1 though it is a bit of legwork.

If you do not want to do it then I can do some work to write a script that does. I would need and example of the spreadsheet, ideally is CSV form.
